app/routers/organizations.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import APIRouter
+from fastapi_pagination.limit_offset import LimitOffsetPage
+
+from app.collections import OrganizationCollection
+from app.db import DBSession
+from app.models import OrganizationRead
+from app.repositories import OrganizationRepository
+
+router = APIRouter()
+
+
+@router.get("/", response_model=LimitOffsetPage[OrganizationRead])
+async def get_organizations(session: DBSession):
+    organizations = OrganizationCollection(repository=OrganizationRepository(session=session))
```

Comment:
Trying to understand the pattern better (specifically the separation between repository and collection) -- I can't think of an example where we would want a collection with different repository (e.g. `EntitlementCollection(OrganizationRepository(session))`). Is that something that can happen? If not, then we can change the collection's `__init__` method to take a session as an argument and initialise the repository itself?

Reply:
as we agreed in the call i will remove this collection layer